pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-ssot"
-version = "2.0.2"
+version = "2.0.3"
```

Comment:
Please revert this. We increment the version when we do the release.

Reply:
Changed